bin/migrate-oats-data/noi/notice_of_intent_init.py

```diff
@@ -9,6 +14,8 @@
 def init_notice_of_intents(conn=None, batch_size=10000):
     logger.info(f"Start {etl_name}")
     with conn.cursor() as cursor:
+        exclusion_table_create(cursor, conn)
```

Comment:
Will this cause an issue when running the following: python migrate.py import? Will it introduce duplicates of records to exclude or fail?